torch/_inductor/fx_passes/quantization.py

```diff
@@ -3157,6 +3103,223 @@ def __init__(
                 )
 
 
+def _register_qlinear_binary_fusion():
```

Comment:
this is really complicated, any thoughts on making it simpler? or is this something you plan to explore in the future?

Reply:
I agree. However, we do find these patterns. This is also the case for other ops. We have all these cases covered in UT. Do you have any suggestions? Thanks.

Reply:
I talked about my suggestion before of getting a pattern by tracing: https://github.com/pytorch/pytorch/blob/90e81a157ae83a3f1684b0977efe849e0e6137ca/torch/ao/quantization/pt2e/utils.py#L339 but I'm not sure if it is applicable here, maybe people from inductor team might have more suggestion here. cc @eellison

Reply:
Thanks for the pointer. However, I feel that we still need to list all possibilities, at module level or aten op level. Maybe we can revisit this later if we find a way to simplify it?

Reply:
yeah sounds good

Reply:
Thanks.